docs/source/deep_dives/comet_logging.rst

```diff
@@ -0,0 +1,53 @@
+.. _comet_logging:
+
+================
+Logging to Comet
+================
+
+This deep-dive will guide you through how to set up logging to Comet in torchtune.
+
+.. grid:: 1
+
+    .. grid-item-card:: :octicon:`mortar-board;1em;` What this deep-dive will cover
+
+      * How to get started with Comet
+      * How to use the :class:`~torchtune.utils.metric_logging.CometLogger`
+      * How to log configs, metrics, and model checkpoints to Comet
+
+Torchtune supports logging your training runs to `Comet <https://www.comet.com/site/?utm_source=torchtune&utm_medium=docs&utm_content=docs>`_.
+An example Comet workspace from a torchtune fine-tuning run can be seen in the screenshot below.
+
+.. image:: ../_static/img/comet_torchtune_project.png
+  :alt: torchtune workspace in Comet
+  :width: 100%
+  :align: center
+
+.. note::
+
+  You will need to install the :code:`comet_ml` package to use this feature.
+  You can install it via pip:
```

Comment:
did you intend to add an example pip command?

Reply:
Good point, added it